docs/release-process.md

```diff
@@ -8,73 +24,38 @@
-    ongoing basis between the releases. Check the CI status for your branch
-    periodically and resolve the failures ASAP. See more in
-    https://wiki.php.net/rfc/travis_ci.
+    Check the CI status for your branch periodically and resolve the failures ASAP. 
+    
+    See more in https://wiki.php.net/rfc/travis_ci.
 
- 4. Ensure that Windows builds will work before packaging.
-
- 5. Follow all steps to the letter. When unclear ask previous RM's (David,
-    Julien, Johannes, Stas, Derick, Ilia, Sara, Remi, or Christoph) before
+ 4. Follow all steps to the letter. When unclear ask previous RM's before
     proceeding. Ideally make sure that for the first releases one of the
     previous RM's is around to answer questions. For the steps related to the
     php/QA/bug websites try to have someone from the webmaster team (Bjori) on
     hand.
 
- 6. Verify the tags to be extra sure everything was tagged properly.
-
- 7. Moving extensions from/to PECL requires write access to the destination.
-    Most developers should have this.
-
-    Moving extensions from php-src to PECL:
+ 5. Verify the tags to be extra sure everything was tagged properly.
 
-    * Ask someone with Git admin access to create a new empty repository GitHub.
-
-    * Clone a new copy of the php-src repository (it will rewrite history, keep
-      contributors commits and include only the extension folder):
-
-        ```sh
-        git clone git@github.com:php/php-src.git ext-name
-        cd ext-name
-        git filter-branch --prune-empty --subdirectory-filter ext/ext-name master
-        ```
-
-    * Set remote Git push URL for the PECL extension:
-
-        ```sh
-        git remote set-url origin git@github.com:php/pecl-ext-name.git
-        ```
-
-    * Create branch and tags structure appropriate for the extension and push:
-
-        ```sh
-        git push -u origin master
-        ```
-
-    If the extension is still usable or not dead, in cooperation with the
-    extension maintainers if any:
-
-    * Create the pecl.php.net/foo package and its content, license, maintainer
-    * Create the package.xml, commit
-    * Release the package
-
-    For moving extensions from PECL to php-src the procedure needs to go through
-    the RFC process and a new Git commit without rewriting the php-src Git
-    commit history.
-
- 8. There is a PHP release Docker image https://github.com/sgolemon/php-release
+ 6. There is a PHP release Docker image https://github.com/sgolemon/php-release
     with forks available to help releasing.
 
-## Rolling a non stable release (alpha/beta/RC)
+ 7. Communication with previous release managers should be conducted via
+    release-managers@php.net
+  
+ 8. References to repositories in this document refer to the canonical source 
+    located at https://github.com/php
+
+## Packaging a non stable release (alpha/beta/RC)
 
  1. Check Windows snapshot builder logs https://windows.php.net/downloads/snaps/
```

Comment:
I think it's better to replace this with a reference to AppVeyor CI (https://ci.appveyor.com/project/php/php-src/history) or else to drop it. There are no more Windows snapshot builds for PHP 8.0+.

Reply:
I just dropped it, seems most appropriate ... we have AV in notes under CI now.